llarp/constants/version.hpp

```diff
@@ -33,5 +37,5 @@ struct Version
 {
   static const char LLARP_NET_ID[];
 };
-
+#endif
```

Comment:
that `#endif` should be a bit higher up so we have the Version struct and netid in windows

Reply:
it doesn't affect normal builds, hence `#ifdef RC_INVOKED`
when `windres(1)` or `rc(1)` run, `/lib/cpp` or rc's internal pre-ASA cpp define the symbol `RC_INVOKED`, which allows you to hide anything that RC may not understand
this is because its internal cpp dates back to the '80s and predates ASA C

Reply:
oh